scripts/street_sign_recognizer.py

```diff
@@ -43,19 +80,24 @@ def sign_bounding_box(self):
         (left_top, right_bottom) where left_top and right_bottom are tuples of (x_pixel, y_pixel)
             defining topleft and bottomright corners of the bounding box
         """
-        # TODO: YOUR SOLUTION HERE
-        left_top = (200, 200)
-        right_bottom = (400, 400)
+        image = deepcopy(self.binary_image)
+        contours = cv2.findContours(image, 1, 2)
```

Comment:
It would be nice to have a couple more comments here, describing what you're trying to do here!   I "think" you are trying to find the largest contour, so that you can then use that to define the sign bounding box.  But it's not clear to someone not already familiar with this method.

Also, `cv2.findContours(image, 1, 2)` is not a very readable line, since I have no clue what the values 1 and 2 do! Reading the docs, it looks like those are the "contour retrieval mode, contour approximation method."

For parameters like that, it's usually better to use the more human readable flags that openCV gives you, i.e.

```python
im2, contours, hierarchy = cv2.findContours(thresh,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE)
```